source/_integrations/watson_tts.markdown

```diff
@@ -29,6 +29,7 @@ To configure Watson TTS, add the following lines to your `configuration.yaml`:
 # Example configuration.yaml entry
 tts:
   - platform: watson_tts
+    watson_url: YOUR_PERSONAL_API_ENDPOINT
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Okay, thanks for the heads up.